.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5502
+}
```

Comment:
You can put these type of files in gitignore

Reply:
Otherwise good to go

Reply:
thank you, i will make the changes